doc/sphinx/users-guide/vcl-built-in-subs.rst

```diff
@@ -356,6 +356,10 @@ The `vcl_backend_fetch` subroutine may terminate with calling
   ``fetch``
     Fetch the object from the backend.
 
+  ``error(status code, reason)``
+    Abandon the backend request and control will be passed to
```

Comment:
I'm not sure we want to use the word "Abandon" here, it's confusing since we already have `return (abandon)`. We can use the same wording as `synth` instead.

Reply:
Got it, check the new force push